primer.html

```diff
@@ -502,12 +505,32 @@ <h2>Updating ShapeTree Instances <a class="cross" href="spec#post-managed"></a><
     tree:shapeTreeInstanceRoot &lt;&gt;.
           </code></pre>
     </section>
-    <!-- ul>
-        <li><b>PhotoAlbum</b> - collects photos or references to photos into sharable albums (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbumShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbumShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/album/PhotoAlbum.shex">schema</a>).</li>
-        <li><b>MultiCal</b> - migrates calendar applications between two common formats, "Common" (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/CalendarShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/CalendarShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/Calendar.shex">schema</a>), and "Google" (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/cal/GoogleCalendar.shex">schema</a>).</li>
-        <li><b>GitHub</b> - a storage repository associating users or groups with repositories, issues and events (ShapeTree <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghShapeTree.ttl">Turtle</a> / <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghShapeTree.jsonld">JSON-LD</a>, <a href="https://github.com/janeirodigital/shapetrees/blob/master/solidApps/staticRoot/gh/ghSchema.shex">schema</a>).</li>
-      </ul -->
 
+    <section id="describing">
+      <h2>Describing Shape Trees <a class="cross" href="spec#describing-shapetrees"></a></h2>
+      <p>
+        To aid in the human-readability of Shape Trees, a SKOS graph can be used to provide a textual representation of the structure.
+      </p>
+      <p>
+        Making use of the <code>tree:step</code> predicate, labels can be applied to specific steps in a Shape Tree using any language.
+        A SKOS index document allows for discovery of multiple definitions of a Shape Tree supporting multiple language preferences and cognitive abilities.
+      </p>
+      <p>
+        A sample SKOS graph defining the terms in the CommonNotes Shape Tree:
+      </p>
+      <pre><code class="example footprint turtle">
+@prefix f: &lt;/data/CommonNotesShapeTree#container&gt; .
+@prefix tree: &lt;http://www.w3.org/ns/shapetree#&gt; .
+@prefix skos: &lt;http://www.w3.org/2004/02/skos/core#&gt; .
+@prefix skosxl: &lt;http://www.w3.org/2008/05/skos-xl#&gt; .
+
```

Comment:
What would this document look like if we added spanish. What about when we have a different character-set (i.e. cyrillic, chinese)?

What would this document look like if we needed to account for a different cognitive ability?

Given that we justify the need for an index based on these two points, I think we need to demonstrate them.

Reply:
Added an example in Russian of same shape tree